test/models/authorization/repository_authorization_test.rb

```diff
@@ -82,6 +82,13 @@ def test_readable_with_versions
       assert_includes Repository.readable, @fedora_17_x86_64_dev
     end
 
+    def test_readable_with_environment
+      refute_includes Repository.readable, @fedora_17_x86_64
+      setup_current_user_with_permissions(:name => "view_lifecycle_environments", :search => "name = \"#{@fedora_17_x86_64.environment.name}\"")
+      assert_includes Repository.readable, @fedora_17_x86_64
```

Comment:
Based on the recent PR with repo description I think we can reduce the last two assertions here to be only:
```
assert_equal Repository.readable, [@fedora_17_x86_64]
```